gitlens/gitlens-account.md

```diff
@@ -9,6 +9,14 @@ taxonomy:
 
 Whether a solo user on a personal license, or an owner managing multiple teams across organizations, learn how to manage your GitKraken GitLens account(s), subscription(s), and organization(s).
 
+The first question to answer: do I need an account to use GitLens or GitLens+? 
+
++ GitLens is COMPLETELY FREE, no account is required, and can be used on any local repository.
++ GitLens+ features on local & public repositories are also COMPLETELY FREE.
++ GitLens+ features on private repositories require a Pro plan (or greater), view pricing [here](https://www.gitkraken.com/gitlens/pricing).
+
+If you are working on GitLens+ on private repositories, you will want to create an acount and purchase a license. See below for more information.
```

Comment:
This is oddly worded. Maybe something like this?
> If you are working ~on~ with GitLens+ features on private repositories, you will ~want~ need to create...

Reply:
yeah, sounds better